src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketDelete.java

```diff
@@ -126,22 +128,23 @@ public void run(ActionContext context) throws Exception {
           deleteCount++;
         } catch (IOException e) {
           LOG.warn(
-            String.format("Failed to delete path '%s'", gcsPath)
+            String.format("Failed to delete path '%s'", gcsPathStr)
           );
         }
       }
     }
 
-    for (GCSPath gcsPath : gcsPathsWild) {
-      String regex = ("\\Q" + gcsPath.getUri().toString() + "\\E").replace("*", "\\E[^/]*\\Q") + "(/.*)?";
+    for (String gcsPathStr : gcsPathsWild) {
+      String regex = ("\\Q" + gcsPathStr + "\\E").replace("*", "\\E[^/]*\\Q") + "(/.*)?";
```

Comment:
Please add a comment indicating what this regex matches

Reply:
Thank you for reviewing. Done adding comment :)